README.md

```diff
@@ -29,10 +29,13 @@ Preventing bugs
 -   validate-nested-of-array-should-set-each
 -   validated-non-primitive-property-needs-type-decorator
 -   all-properties-are-whitelisted
+-   no-conflicting-decorators
 
 Security
 
 -   should-specify-forbid-unknown-values
+-   all-defined-property-must-be-decored
```

Comment:
i wouldn't say these are security based

new rules should have an explanation also. These ones would need instructions on how to turn on also if not in the recommended set.